app/livechat/server/api/v1/room.js

```diff
@@ -103,7 +103,9 @@ API.v1.addRoute('livechat/room.transfer', {
 			// update visited page history to not expire
 			Messages.keepHistoryForToken(token);
 
-			if (!Promise.await(Livechat.transfer(room, guest, { roomId: rid, departmentId: department }))) {
+			const transferedBy = { _id: room.servedBy._id, username: room.servedBy.username };
```

Comment:
Through this endpoint, the `transfer` process is requested from the `visitor`, not the current `room agent`.